pkg/query-service/model/alerting.go

```diff
@@ -18,6 +18,7 @@ const (
 	StateFiring
 	StateNoData
 	StateDisabled
+	StateRecovering
```

Comment:
Why is this `StateRecovering` give the highest position? Shouldn't be lower than the `StateFiring`. `r.State` returns the overall status of the alert. If there is a multi-dimensional alert, where one series is recovering and rest are firing, should the overall status be "firing" or "recovering"?